ui fails to start locally

```
lamport ~/p/taskcluster/ui [main] $ rm -rf node_modules/
yarn
lamport ~/p/taskcluster/ui [main] $ yarn
yarn install v1.22.5
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.1: The platform "linux" is incompatible with this module.
info "fsevents@2.1.1" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > @material-ui/pickers@3.2.7" has unmet peer dependency "@date-io/core@^1.3.6".
warning " > material-ui-json-schema-viewer@1.0.3" has incorrect peer dependency "react@^16.12.0".
warning " > material-ui-json-schema-viewer@1.0.3" has incorrect peer dependency "react-dom@^16.12.0".
warning " > react-apollo@3.1.5" has unmet peer dependency "@types/react@^16.8.0".
warning "react-apollo > @apollo/react-common@3.1.4" has unmet peer dependency "@types/react@^16.8.0".
warning "react-apollo > @apollo/react-components@3.1.5" has unmet peer dependency "@types/react@^16.8.0".
warning "react-apollo > @apollo/react-components@3.1.5" has unmet peer dependency "apollo-cache@^1.3.2".
warning "react-apollo > @apollo/react-hoc@3.1.5" has unmet peer dependency "@types/react@^16.8.0".
warning "react-apollo > @apollo/react-hooks@3.1.5" has unmet peer dependency "@types/react@^16.8.0".
warning "@mozilla-frontend-infra/react-lint > @neutrinojs/airbnb@9.0.0-rc.4" has incorrect peer dependency "neutrino@9.0.0-rc.4".
warning "@mozilla-frontend-infra/react-lint > @neutrinojs/airbnb > @neutrinojs/eslint@9.0.0-rc.4" has incorrect peer dependency "neutrino@9.0.0-rc.4".
warning "@neutrinojs/airbnb > eslint-config-airbnb@18.0.1" has incorrect peer dependency "eslint-plugin-react-hooks@^1.7.0".
warning "react-styleguidist > webpack-dev-server@2.11.3" has incorrect peer dependency "webpack@^2.2.0 || ^3.0.0".
warning "react-styleguidist > webpack-dev-server > webpack-dev-middleware@1.12.2" has incorrect peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
[5/5] Building fresh packages...
Done in 36.93s.
lamport ~/p/taskcluster/ui [main] $ yarn start
yarn run v1.22.5
$ cross-env GENERATE_ENV_JS=1 webpack-dev-server --mode development
ℹ ｢wds｣: Project is running at http://localhost:5080/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /home/dustin/p/taskcluster/ui
ℹ ｢wds｣: 404s will fallback to /index.html
✖ ｢wdm｣: Time: 53135ms

ERROR in ./node_modules/ssh2/lib/agent.js
Module not found: Error: Can't resolve 'child_process' in '/home/dustin/p/taskcluster/ui/node_modules/ssh2/lib'

ERROR in ./node_modules/ssh2/lib/client.js
Module not found: Error: Can't resolve 'dns' in '/home/dustin/p/taskcluster/ui/node_modules/ssh2/lib'

ERROR in ./node_modules/ssh2/lib/server.js
Module not found: Error: Can't resolve 'net' in '/home/dustin/p/taskcluster/ui/node_modules/ssh2/lib'

ERROR in ./node_modules/ssh2/lib/agent.js
Module not found: Error: Can't resolve 'net' in '/home/dustin/p/taskcluster/ui/node_modules/ssh2/lib'

ERROR in ./node_modules/ssh2/lib/client.js
Module not found: Error: Can't resolve 'net' in '/home/dustin/p/taskcluster/ui/node_modules/ssh2/lib'
ℹ ｢wdm｣: Failed to compile.
ℹ ｢wdm｣: Compiling...
```

--- a/changelog/issue-3543.md
+++ b/changelog/issue-3543.md
@@ -1,0 +1,4 @@
+audience: general
+level: silent
+reference: issue 3543
+---

--- a/ui/package.json
+++ b/ui/package.json
@@ -42,7 +42,7 @@
     "deep-sort-object": "^1.0.2",
     "deepmerge": "^4.0.0",
     "dexie": "^2.0.4",
-    "docker-exec-websocket-server": "^1.4.0",
+    "docker-exec-websocket-client": "^1.4.0",
     "dot-prop-immutable": "^1.4.0",
     "escape-string-regexp": "^2.0.0",
     "fast-memoize": "^2.5.1",

--- a/ui/src/components/Shell/index.jsx
+++ b/ui/src/components/Shell/index.jsx
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import { string } from 'prop-types';
 import { hterm, lib } from 'hterm-umd';
-import { DockerExecClient } from 'docker-exec-websocket-server';
+import { DockerExecClient } from 'docker-exec-websocket-client';
 import { dial } from 'ws-shell';
 import withAlertOnClose from '../../utils/withAlertOnClose';
 

